deploy/helm/sumologic/values.yaml

```diff
@@ -1520,37 +1520,6 @@ kube-prometheus-stack:
   ## Labels to apply to all kube-prometheus-stack resources
   commonLabels: {}
 
-  defaultRules:
-    rules:
-      alertmanager: false
-      etcd: false
-      configReloaders: false
-      general: false
-      k8s: false
-      kubeApiserverAvailability: false
-      kubeApiserverBurnrate: false
-      kubeApiserverHistogram: false
-      kubeApiserverSlos: false
-      kubeControllerManager: false
-      kubelet: false
-      kubeProxy: false
-      kubePrometheusGeneral: false
-      kubePrometheusNodeRecording: true
-      kubernetesApps: false
-      kubernetesResources: false
-      kubernetesStorage: false
-      kubernetesSystem: false
-      kubeSchedulerAlerting: false
-      kubeSchedulerRecording: false
-      kubeStateMetrics: false
-      network: false
-      node: true
-      nodeExporterAlerting: false
-      nodeExporterRecording: false
-      prometheus: false
-      prometheusOperator: false
-      windows: false
-
```

Comment:
These are all enabled by default, so what you need to do here instead is to set everything to `false`.

Reply:
set all of them to `false`, thanks